app/assets/javascripts/scp-infercnv-ideogram.js

```diff
@@ -99,7 +99,7 @@ function updateThreshold(event) {
 
 /** Create slider to adjust expression threshold for "gain" or "loss" calls */
 function addThresholdControl() {
-  if (typeof(expressionThreshold) === 'undefined') {
+  if (typeof window.expressionThreshold === 'undefined') {
```

Comment:
this variable is only accessed in this file, so a window variable seems like overkill.  Add it to the `let` declarations at the top of this file?  Or if that doesn't work (I don't know the fine details of how scoping works in stuff that's required in by the sprockets pipeline), at least scope this to window.SCP

Reply:
Good call with `let` -- done in fda6d7a.